docs/elasticsearch/cluster.md

```diff
@@ -21,13 +21,17 @@ This is a four-node cluster that has one dedicated master node, one dedicated co
 
 The following table provides brief descriptions of the node types.
 
-Node type | Description
-:--- | :--- |
-`Master` | Manages the overall operation of a cluster and keeps track of the cluster state.
-`Master-eligible` | Available for election to become a new master node.
-`Data` | Stores and searches data.
-`Ingest` | Preprocesses data before storing it in a cluster.
-`Coordinating` | Routes search and aggregation requests.
+Node type | Description | Best practices for production
+:--- | :--- | :-- |
+`Master` | Manages the overall operation of a cluster and keeps track of the cluster state. This includes creating and deleting indices, keeping track of the nodes that join and leave the cluster, checking the health of each node in the cluster (by running ping requests), and allocating shards to nodes. | Three dedicated master nodes in three different zones is right for all/just about all production use cases. This to make sure your cluster never loses quorum. Two of them will be idle for most of the time except when one of them goes down or needs some maintenance.
+`Master-eligible` | Elects one node among them as the master node through a voting process. | Make sure you have dedicated master nodes. The way to achieve a dedicated node type is to mark all other node types as false. In this case, you have to mark all the other nodes as not master-eligible.
+`Data` | Stores and searches data. Performs all data-related operations (indexing, searching, aggregating) on local shards. These are the worker nodes of your cluster and need more disk space than any other node type. | Keep adding data nodes in multiples of three across all your zones proportional to the volume of your data. We recommend using storage and RAM-heavy nodes.
```

Comment:
"Keep adding data nodes in multiples of three across all your zones proportional to the volume of your data."

I know what you're saying here, but I don't think it'll be clear to most people. Something like, "As you add data nodes, keep them balanced between zones. For example, if you have three zones, add data nodes in multiples of three, one for each zone."